src/handlers/localstorage/accountLocal.js

```diff
@@ -205,6 +211,19 @@ export const saveLocalTransactions = (transactions, accountAddress, network) =>
     transactionsVersion
   );
 
+/**
+ * @desc save transaction signatures
+ * @param  {Object}   [transactionSignatures]
+ */
+export const saveLocalTransactionSignatures = transactionSignatures =>
```

Comment:
This file is intended for data that is per wallet account, so let's not mix up this pattern.
For global data, you can either create a new file under `handlers/localstorage` or use `handlers/localstorage/globalSettings`